apps/music/js/music.js

```diff
@@ -128,7 +134,13 @@ var App = (function() {
     titleElement.dataset.l10nId = l10nIds.title;
     textElement.dataset.l10nId = l10nIds.text;
 
-    document.getElementById('overlay').classList.remove('hidden');
+    Array.forEach(document.body.children, function(elt) {
+      if (elt.id === 'overlay') {
+        elt.classList.remove('hidden');
+      } else {
+        elt.setAttribute('aria-hidden', true);
+      }
+    });
```

Comment:
@eeejay These two logics(Above and down) looks the same but opposite, do you think it's nice to use a function then pass different parameters? I am fine with the current code, just want to make these ui logics more organized.